geomesa-arrow/geomesa-arrow-gt/src/main/scala/org/locationtech/geomesa/arrow/vector/SimpleFeatureVector.scala

```diff
@@ -56,7 +56,12 @@ class SimpleFeatureVector private [arrow] (
     */
   def clear(): Unit = underlying.setValueCount(0)
 
-  override def close(): Unit = CloseWithLogging.raise(Seq(underlying) ++ allocator)
+  override def close(): Unit = {
+    CloseWithLogging.raise(Seq(underlying))
+    this.synchronized {
```

Comment:
i think we don't need this any more, right? if so, we should try to make sure it gets called even if the first call fails

Reply:
Yeah, I think it can go.  I'll do that.